src/__mocks__/pages/api/puzzles/instances/create.ts

```diff
@@ -0,0 +1,42 @@
+import * as faker from 'faker';
+import { DIFFICULTY } from '../../../../../types/global';
+
+const DIFFICULTY_LIST = ['EASY', 'MEDIUM', 'HARD'];
```

Comment:
hmm, this creates 2 points of truth, but I know why you did it, leave a TODO so later we can figure out how to ensure this list is the same as the type definition

Reply:
Sure!